common/components/Forms/AddServiceForm/index.tsx

```diff
@@ -17,18 +16,36 @@ interface Props {
 
 const AddServiceForm: React.FC<Props> = ({ form, edit, currentService }) => {
   const { MonthPicker } = DatePicker
-  const initialValues = useInitialValues(currentService)
-  console.log(currentService)
+  const [initialValues, setInitialValues] = useState<any>({})
 
   useEffect(() => {
     if (currentService) {
-      form.setFieldsValue(initialValues)
+      const newInitialValues = {
+        domain: currentService?.domain?.name || '',
+        street:
+          (currentService?.street &&
+            `${currentService.street.address} (м. ${currentService.street.city})`) ||
+          '',
+        date: currentService?.date ? dayjs(currentService?.date) : null,
+        electricityPrice: currentService?.electricityPrice || 0,
+        inflicionPrice: currentService?.inflicionPrice || 0,
+        rentPrice: currentService?.rentPrice || 0,
+        waterPrice: currentService?.waterPrice || 0,
+        description: currentService?.description || '',
+        waterPriceTotal: currentService?.waterPriceTotal || 0,
+        garbageCollectorPrice: currentService?.garbageCollectorPrice || 0,
+        domainId: currentService?.domain?._id?.toString() || '',
+        streetId: currentService?.street?._id?.toString() || '',
+      }
+
+      setInitialValues(newInitialValues)
+      form.setFieldsValue(newInitialValues)
     }
-  }, [currentService, form, initialValues])
+  }, [currentService, form])
 
   const serviceData = {
-    domain: currentService?.domain?._id?.toString(),
-    street: currentService?.street?._id?.toString(),
+    domain: initialValues.domainId,
+    street: initialValues.streetId,
   }
```

Comment:
This is unwanted values, get rid of this, if possible. Instead collect values directly from `form` with `Form.useWatch('name', form)`